src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,61 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+
+    private final static String DEFAULT_SPLITTER_START = "[,";
+    private final static String DEFAULT_SPLITTER_END = ":]";
+    private final static int REGEX_SPLITTER_GROUP = 1;
+    private final static int REGEX_STRING_INPUT_GROUP = 2;
+    private final static String STRING_ZERO_NUMBER = "0";
+    private final static String REG_EXPRESSION = "//(.)\n(.*)";
+
+    private String splitter;
+    private String targetString;
+
+    private List<Number> numbers = new ArrayList<>();
+
+    private StringAddCalculator(String splitter, String targetString) {
+        this.splitter = splitter;
+        this.targetString = targetString;
+    }
+
+    public static StringAddCalculator execute(String inputValue) {
+        if (inputValue == null || inputValue.isBlank()) {
+            return new StringAddCalculator(DEFAULT_SPLITTER_START + DEFAULT_SPLITTER_END, STRING_ZERO_NUMBER);
+        }
+
+        Matcher matcher = Pattern.compile(REG_EXPRESSION).matcher(inputValue);
```

Comment:
Pattern.compile() 내부 구현을 보면 
```
// Pattern
public static Pattern compile(String regex) {
        return new Pattern(regex, 0);
}
```
execute() 메서드를 호출할때마다 String을 패턴 매칭을 위해
매번 성능이 비싼 Pattern 인스턴스를 생성하고 Pattern 인스턴스는 한번 쓰고 버려지기 때문에
성능적으로 좋지 않습니다. 뿐더러 자바 doc에서도 한번 컴파일 후 재사용 할것을 권장하고 있습니다.

If a pattern is to be used multiple times, compiling it once and reusing
it will be more efficient than invoking this method each time

매번 불필요한 객체 생성을 피하도록 개선하면 좋을 것 같아요 :)